src/ImageSharp.Drawing/Processing/TextGraphicsOptions.cs

```diff
@@ -9,120 +10,151 @@ namespace SixLabors.ImageSharp.Processing
     /// <summary>
     /// Options for influencing the drawing functions.
     /// </summary>
-    public struct TextGraphicsOptions
+    public class TextGraphicsOptions
     {
-        private const int DefaultTextDpi = 72;
+        private int antialiasSubpixelDepth = 16;
+        private float blendPercentage = 1F;
+        private float tabWidth = 4F;
+        private float dpiX = 72F;
+        private float dpiY = 72F;
 
         /// <summary>
-        /// Represents the default <see cref="TextGraphicsOptions"/>.
+        /// Gets the default <see cref="TextGraphicsOptions"/> instance.
         /// </summary>
-        public static readonly TextGraphicsOptions Default = new TextGraphicsOptions(true);
-
-        private float? blendPercentage;
-
-        private int? antialiasSubpixelDepth;
-
-        private bool? antialias;
-
-        private bool? applyKerning;
-
-        private float? tabWidth;
-
-        private float? dpiX;
-
-        private float? dpiY;
-
-        private HorizontalAlignment? horizontalAlignment;
-
-        private VerticalAlignment? verticalAlignment;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="TextGraphicsOptions" /> struct.
-        /// </summary>
-        /// <param name="enableAntialiasing">If set to <c>true</c> [enable antialiasing].</param>
-        public TextGraphicsOptions(bool enableAntialiasing)
-        {
-            this.applyKerning = true;
-            this.tabWidth = 4;
-            this.WrapTextWidth = 0;
-            this.horizontalAlignment = HorizontalAlignment.Left;
-            this.verticalAlignment = VerticalAlignment.Top;
-
-            this.antialiasSubpixelDepth = 16;
-            this.ColorBlendingMode = PixelColorBlendingMode.Normal;
-            this.AlphaCompositionMode = PixelAlphaCompositionMode.SrcOver;
-            this.blendPercentage = 1;
-            this.antialias = enableAntialiasing;
-            this.dpiX = DefaultTextDpi;
-            this.dpiY = DefaultTextDpi;
-        }
+        public static TextGraphicsOptions Default { get; } = new TextGraphicsOptions();
```

Comment:
The singleton is still there. It's completely unnecessary IMO. Shared mutable state that is quite arbitrary. We can replace it with `new TextGraphicsOptions()` on all call sites. Same for `GraphicsOptions`.

Reply:
Ah, didn't get your point previously. Removed.